js/build.js

```diff
@@ -152,6 +152,12 @@ Fliplet.Widget.instance('chat', function (data) {
   var customWidth = 1024;
   var customHeight = 1024;
 
+  if (Fliplet.Navigate.query.conversationId) {
+    Fliplet.UI.Toast('Opening channel...');
```

Comment:
You should use:

```js
Fliplet.UI.Toast({
  message: 'Opening channel...',
  duration: false
});
```

...to prevent the Toast dismissed after 2 seconds (in case it takes longer than 2 seconds for the conversation to load.

When it is loaded, you'll then want to call:

```js
Fliplet.UI.Toast.dismiss();
```

To take it to the next level, you could be calling it like:

```js
var requestCanceled = false;
Fliplet.UI.Toast({
  message: 'Opening channel...',
  duration: false,
  backdrop: true,
  tapToDismiss: false,
  position: 'center',
  actions: [{
    label: 'Cancel',
    action: function () {
      requestCanceled = true; // Use this flag to stop further actions
    }
  }]
});
```

Reply:
@tonytlwu fixed